source/clear-linux/tutorials/kata_migration.rst

```diff
@@ -0,0 +1,86 @@
+.. _kata_migration:
+
+Clear Containers to Kata Containers\* migration
+###############################################
+
+`Clear Containers`_ and `Kata Containers`_ can co-exist in the same system; at the moment both can be installed through the :file:`containers-virt bundle`. However, we recommend that you migrate to Kata Containers because Clear Containers is no longer maintained_ and will soon be deprecated on |CL|.
+
+.. _Clear Containers: https://github.com/clearcontainers
+.. _Kata Containers: https://github.com/kata-containers
+.. _maintained: https://github.com/kata-containers/documentation/blob/master/Upgrading.md#maintenance-warning
+
+To switch to Kata Containers in |CL|, complete the following:
+
+Stop Clear Containers instances
+*******************************
+
+Before making any changes, all running Clear Containers must be stopped:
+
+.. code-block:: bash
+
+    for container in $(sudo docker ps -q); do sudo docker stop $container; done
+
+.. note::
+
+    This step assumes that Clear Containers is on a Docker\* system.
+
+Manually migrate customized configuration files
+***********************************************
+
+There is no way to do an automatic migration of customized configuration files. If any change was made on the `Clear Containers configuration`_ which is also required on the `Kata Containers configuration`_, you must make such changes manually. 
+Make any required changes before going any further, because all Clear Containers configuration files will eventually be removed.
+
+.. _Clear Containers configuration: https://github.com/clearcontainers/runtime#configuration
+.. _Kata Containers configuration: https://github.com/kata-containers/runtime#configuration
+
+No need to remove Clear Containers packages
+*******************************************
+We recommend that you migrate to use Kata Containers as soon as possible before all Clear Containers packages are removed. You do not need to manually remove any Clear Containers packages.
+
+Disable Clear Containers manager configuration
+**********************************************
```

Comment:
@MCamp859. Could you please apply these changes **from this section to the end** of the tutorial?: 

- Fix the indentation in .rst syntax to correct formatting on these steps (errors occur)
- Revise commands to second person instead of passive tense (where necessary)